src/components/common/Button/index.js

```diff
@@ -1,7 +1,5 @@
 import { func, bool, string } from 'prop-types';
 
-import './styles.css';
-
 const Button = ({ children, type = 'button', disabled, handleClick }) => (
```

Comment:
I'm not sure what are the possible options for the type. Maybe adding a type would help. What do you think?

Reply:
This is part of the template to

Reply:
You're right, I assume it was TS, force of habit hahaha